src/ucp/api/ucp_compat.h

```diff
@@ -33,4 +33,17 @@ void ucp_request_release(void *request);
 void ucp_ep_destroy(ucp_ep_h ep);
 
 
+/**
+ * @ingroup UCP_ENDPOINT
+ * @deprecated Replaced by @ref ucp_tag_recv_request_test and
+ *             @ref ucp_request_check_status depends on use case.
+ *
+ * @note Please, use @ref ucp_request_check_status in cases then only check
+ *       status is needed for any type of request and
+ *       @ref ucp_tag_recv_request_test for request returned from
+ *       @ref ucp_tag_recv_nb routine and out-parameter @a info is required.
+ */
```

Comment:
Suggest re-wording:
Please use ucp_request_check_status for cases that only need to check the completion status of an outstanding request. ucp_request_check_status can be used for any type of request. ucp_tag_recv_request_test should only be used for requests returned by ucp_tag_recv_nbr for which additional information (returned via the _info_ pointer) is needed.